code/game/gamemodes/vampire/vampire.dm

```diff
@@ -5,6 +5,15 @@
 	var/list/datum/mind/vampire_enthralled = list() //those controlled by a vampire
 	var/list/vampire_thralls = list() //vammpires controlling somebody
 
+
+
+/mob/living/carbon/human/Life()
+	. = ..()
+	if(is_vampire(src))
+		var/datum/antagonist/vampire/vamp = mind.has_antag_datum(/datum/antagonist/vampire)
+		vamp.vampire_life()
```

Comment:
why is this `vampire_life`. Why is this not `antag_life` or something, and it's called in a loop inside `human/life.dm`

Reply:
i can redo the proc yeah. I am going off kinda hippie doing this so. but yeah doing it as a proc would make it eaiser on other antags later if some othere chose to do stuff.